src/lib/isSlug.js

```diff
@@ -4,5 +4,8 @@ let charsetRegex = /^[^-_](?!.*?[-_]{2,})([a-z0-9\\-]{1,}).*[^-_]$/;
 
 export default function isSlug(str) {
   assertString(str);
+  if (str.includes(' ')) {
```

Comment:
I think there's a way this could be put on the regex?

Reply:
Hey, I have created a regex that would reject the spaces at start, in between and at the end, shall I create a PR for the same?